tests/tck/features/insert/InsertIfNotExists.feature

```diff
@@ -168,3 +168,110 @@ Feature: Insert vertex and edge with if not exists
+    When try to execute query:
+      """
+      INSERT VERTEX person VALUES "Tom":("Tom")
+      """
+    Then a SemanticError should be raised at runtime: Column count doesn't match value count.
+    When try to execute query:
+      """
+      INSERT VERTEX person VALUES "Tom":(2, "Tom")
+      """
+    Then a ExecutionError should be raised at runtime: Storage Error: The data type does not meet the requirements. Use the correct type of data.
+    When try to execute query:
+      """
+      INSERT VERTEX person VALUES "Tom":("Tom", "2")
+      """
+    Then a ExecutionError should be raised at runtime: Storage Error: The data type does not meet the requirements. Use the correct type of data.
+    When try to execute query:
+      """
+      INSERT VERTEX person VALUES "Tom":("Tom", 2, "3")
+      """
+    Then a SemanticError should be raised at runtime: Column count doesn't match value count.
+    When executing query:
+      """
+      INSERT VERTEX IF NOT EXISTS
+        person
+      VALUES
+        "Conan":("Conan", 10),
+        "Yao":("Yao", 11),
+        "Conan":("Conan", 11),
+        "Tom":("Tom", 3);
+      """
+    Then the execution should be successful
+    # check vertex result with fetch
+    When executing query:
+      """
+      FETCH PROP ON person "Tom" YIELD person.age as age
+      """
+    Then the result should be, in any order, with relax comparison:
+      | VertexID | age |
+      | "Tom"    | 2   |
+    # check insert edge with default props
+    When executing query:
+      """
+      INSERT EDGE like(likeness) VALUES "Tom"->"Conan":(100)
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      INSERT EDGE like VALUES "Tom"->"Conan":(200)
+      """
+    Then the execution should be successful
+    When executing query:
+      """
+      INSERT EDGE like VALUES "Tom"->"Conan":("200")
+      """
+    Then a ExecutionError should be raised at runtime: Storage Error: The data type does not meet the requirements. Use the correct type of data.
+    When executing query:
+      """
+      INSERT EDGE like() VALUES "Tom"->"Conan":(200)
+      """
+    Then a SemanticError should be raised at runtime: Column count doesn't match value count.
+    When executing query:
+      """
+      INSERT EDGE like VALUES "Tom"->"Conan":(200, 2)
+      """
+    Then a SemanticError should be raised at runtime: Column count doesn't match value count.
+    When executing query:
+      """
+      INSERT EDGE IF NOT EXISTS like VALUES "Tom"->"Conan":(300)
+      """
+    Then the execution should be successful
+    # check edge result with fetch
+    When executing query:
+      """
+      FETCH PROP ON like "Tom"->"Conan" YIELD like.likeness
+      """
+    Then the result should be, in any order:
+      | like._src | like._dst | like._rank | like.likeness |
+      | 'Tom'     | 'Conan'   | 0          | 200           |
```

Comment:
Don't forget to drop the tmp space by `And drop the used space`

Reply:
done